Apps/DebugApp/DebugApp/DebugViewController.swift

```diff
@@ -11,6 +11,7 @@ import Turf
 public class DebugViewController: UIViewController {
 
     internal var mapView: MapView!
+    internal var runningAnimator: CameraAnimator?
```

Comment:
I know this is debug code, but since you are using `makeCameraAnimator` and not `makeRunningCameraAnimator` let's call this `cameraAnimator`

Or you can just remove all changes to the debugVC as a whole

Reply:
we'll be doing that before merging to `main`